src/model/adapter.py

```diff
@@ -79,34 +68,51 @@ def adapt_from_medium(model, medium):
     ])
 
     # Make all metabolites in the medium consumable by setting the exchange reactions lower bound to a negative number
+    medium_mapping = {}
     for compound in medium:
         try:
             metabolite = get_unique_metabolite(model, compound['id'], 'e', 'CHEBI')
```

Comment:
I don't know the definition of this function right now but I'm assuming that `'e'` is supposed to select the extracellular compartment. It'd be safer to detect the extracellular compartment first and use its extension.

Reply:
Yes, the function looks up a metabolite in the given compartment, also using the annotations field to look for the identifier in the given namespace, e.g. chebi.

> It'd be safer to detect the extracellular compartment first and use its extension.

Sorry, it's not clear to me what you mean by that, could you elaborate a bit?